docs/source/api-advanced.rst

```diff
@@ -13,7 +13,6 @@ Before using this chapter's classes, you need to be able to import the following
 .. code-block:: python
```

Comment:
Change line `Before using this chapter's classes, you need to be able to import the following modules.` to something like `... able to import the following module`.

Reply:
good catch